apps/s_client.c

```diff
@@ -1867,14 +1871,24 @@ int s_client_main(int argc, char **argv)
             int foundit = 0;
             BIO *fbio = BIO_new(BIO_f_buffer());
             BIO_push(fbio, sbio);
-            /* wait for multi-line response to end from SMTP */
+            /* Wait for multi-line response to end from LMTP or SMTP */
             do {
                 mbuf_len = BIO_gets(fbio, mbuf, BUFSIZZ);
             }
             while (mbuf_len > 3 && mbuf[3] == '-');
-            BIO_printf(fbio, "EHLO %s\r\n", ehlo);
+            switch ((PROTOCOL_CHOICE) starttls_proto) {
```

Comment:
gaah.  in order to avoid some warngs (switch enum value unused), please rewrite this as ````
if (startttls_proto == (int)PROTO_LMTP)
   BIO_printf(
else
   BIO_printf ...
```

sigh.